requirements.txt

```diff
@@ -6,3 +6,4 @@ pytz==2017.2
 requests>=2.7.0,<3.0
 responses>=0.10.8,<1.0
 schematics==2.0.0.dev2
+pandas>=0.20.0
```

Comment:
The requirements are for the packages you need to run the project. pandas is not a requirement, I'd just add a comment in the README.md of your directory.

Reply:
sounds good!